l10n_ch_import_cresus/models/account_tax.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields
+
+
+class AccountTax(models.Model):
+    _inherit = "account.tax"
+
+    tax_cresus_mapping = fields.Char(
+        size=128,
```

Comment:
I have no idea. Besides, maybe it's only the case on earlier versions, so better drop it.